helix-front/pom.xml

```diff
@@ -135,7 +135,7 @@
               <goal>exec</goal>
             </goals>
             <configuration>
-              <executable>assemble/package.sh</executable>
+              <executable>assemble/assemble.sh</executable>
```

Comment:
renaming this to avoid confusion with `package.json` when hot-switching in VSCode.